README.md

```diff
@@ -1,15 +1,12 @@
-[![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/AY2324S1-CS2103T-T15-4/tp/actions)
+[![CI Status](https://github.com/AY2324S1-CS2103T-T15-4/tp/workflows/Java%20CI/badge.svg)](https://github.com/AY2324S1-CS2103T-T15-4/tp/actions)
+[![codecov](https://codecov.io/gh/AY2324S1-CS2103T-T15-4/tp/graph/badge.svg)](https://codecov.io/gh/AY2324S1-CS2103T-T15-4/tp)
 
 ![Ui](docs/images/Ui.png)
 
-* This is **a sample project for Software Engineering (SE) students**.<br>
-* This project is based on the AddressBook-Level3 project created by the [SE-EDU initiative](https://se-education.org).
-  Example usages:
-  * as a starting point of a course project (as opposed to writing everything from scratch)
-  * as a case study
-* The project simulates an ongoing software project for a desktop application (called _AddressBook_) used for managing contact details.
-  * It is **written in OOP fashion**. It provides a **reasonably well-written** code base **bigger** (around 6 KLoC) than what students usually write in beginner-level SE modules, without being overwhelmingly big.
+* our **team project** for the course **CS2103T** in AY2023/2024 Semester 1.
```

Comment:
Would it be possible to capitalize the "o" of "our"?

Reply:
Thanks Gerald for the heads up! Will edit accordingly